translations/pt_BR.ts

```diff
@@ -0,0 +1,3271 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pt_BR" sourcelanguage="en">
+<context>
+    <name>AppWindow</name>
+    <message>
+        <source>Hot Exit</source>
+        <translation>Saída</translation>
```

Comment:
@crcpucmg Does "Saída" mean "Hot Exit" in Portuguese?

https://www.collinsdictionary.com/dictionary/portuguese-english/saida